libdnf/rpm/transaction.cpp

```diff
@@ -579,13 +579,16 @@ void * Transaction::ts_callback(
             // total is return code - if (!RPMSCRIPT_FLAG_CRITICAL) return_code = RPMRC_OK
             auto script_type = rpm_tag_to_script_type(static_cast<rpmTag_e>(amount));
             auto nevra = trans_element_to_nevra(trans_element);
+            auto return_code = static_cast<rpmRC_e>(total);
```

Comment:
What about to use a different logger level when return_code `RPMRC_OK`. Let say logger.warning() and keep error level when return code is not OK.